src/services/config.js

```diff
@@ -150,12 +161,31 @@ class ConfigData {
     configObj.projects = configObj.projects || [];
     configObj.projects.forEach((project) => this.addProfile(project.id, project.accountSid, project.region));
     this.setActiveProfile(configObj.activeProject);
+    this.profiles = configObj.profiles || {};
   }
 
   sanitize(string) {
     // Trim whitespace if given a non-null string.
     return string ? string.trim() : string;
   }
+
+  addApiKeysToProfile(profile) {
+    if (profile) {
+      const credentials = this.getApiKeysByProfileID(profile.id);
+      if (credentials) {
+        profile.apiKey = credentials.apiKey;
+        profile.apiSecret = credentials.apiSecret;
+      }
+    }
+  }
+
+  getApiKeysByProfileID(profileId) {
+    const { profiles } = this;
+    if (profileId && profiles.hasOwnProperty(profileId)) {
+      return profiles[profileId];
+    }
+    return null;
```

Comment:
Can return `profiles[profileId]` without an explicit check. It'll fall back to `undefined` if key not present